app/models/api_authenticator.rb

```diff
@@ -0,0 +1,42 @@
+class ApiAuthenticator < Struct.new(:controller)
```

Comment:
do you mind of we do that separately? I could create a card for it...

Seems like we might want to use something more detailed like `pundit` which would create a lot of noise in here... or maybe you just thought we should rename to `ApiAuthentcatorPolicy` ? (the latter could def be done in the context of this PR)